recipes-wpe/thunder/include/compositor.inc

```diff
@@ -8,22 +8,28 @@ WPE_COMPOSITOR_HW_READY 	?= "0"
 
 # Additional flags
 WPE_COMPOSITOR_AUTOSTART 	?= "true"
+WPE_COMPOSITOR_BOXMODE		?= "3"
+WPE_COMPOSITOR_IRMODE		?= "16"
+WPE_COMPOSITOR_MEMORY_GFX	?= ""
 WPE_COMPOSITOR_OUTOFPROCESS ?= "true"
 WPE_COMPOSITOR_RESOLUTION	?= "1080p"
-WPE_COMPOSITOR_MEMORY_GFX	?= ""
-WPE_COMPOSITOR_IRMODE		?= "16"
 WPE_COMPOSITOR_SVP			?= "None"
 
+# change to Weston or Westeros, will be included in case of Wayland
+WPE_COMPOSITOR_SUB_IMPLEMENTATION = "Westeros"
```

Comment:
making it case insensitive would be better

Reply:
Yeah this is the way the CMAKE is setup at the other end. We can ofc send a pull request to the main thunder repo to make this lowercase.